lib/stack_master/parameter_resolvers/ami_finder.rb

```diff
@@ -0,0 +1,34 @@
+module StackMaster
+  module ParameterResolvers
+    class AmiFinder
+      def initialize(region)
+        @region = region
+        @owner = ['self']
+      end
+
+      def build_filters(value, prefix = nil)
+        value.split(',').map do |name_with_value|
+          name, value = name_with_value.strip.split('=')
+          @owner.push name if name == 'owner_id'
```

Comment:
Should this really `push`?  If I specified an owner value, I'd expect the results to be restricted to that owner.  Leaving `self` in the list of possible owners might return more than what the caller was expecting.